src/metaTypes.h

```diff
@@ -35,18 +36,18 @@ namespace METAIO_NAMESPACE {
 typedef char                MET_ASCII_CHAR_TYPE;
 typedef char                MET_CHAR_TYPE;
 typedef unsigned char       MET_UCHAR_TYPE;
-typedef short               MET_SHORT_TYPE;
-typedef unsigned short      MET_USHORT_TYPE;
-typedef int                 MET_INT_TYPE;
-typedef unsigned int        MET_UINT_TYPE;
-typedef long                MET_LONG_TYPE;
-typedef unsigned long       MET_ULONG_TYPE;
+typedef int16_t             MET_SHORT_TYPE;
+typedef uint16_t            MET_USHORT_TYPE;
+typedef int32_t             MET_INT_TYPE;
+typedef uint32_t            MET_UINT_TYPE;
+typedef int32_t             MET_LONG_TYPE;
+typedef uint32_t            MET_ULONG_TYPE;
```

Comment:
In ITK, the size of MET_LONG_TYPE is compared to the size of long and int [1].
We could indeed rename MET_INT_TYPE to MET_INT32_TYPE and MET_LONG_TYPE to MET_INT64_TYPE as for the NIFTI format [2], and that would be cleaner, but would remove backward compatibility.
[1] https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/IO/Meta/src/itkMetaImageIO.cxx#L212-L219
[2] https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/ThirdParty/NIFTI/src/nifti/niftilib/nifti1.h#L506-L509